designs/osc_high_availability/OSC_High_Availability.md

```diff
@@ -0,0 +1,83 @@
+# OSC High Availability
+- OSC High Availability eliminates from single points of failure which protects from unplanned downtime.
+- Performs regular checks to make sure each OSC Service Node is working properly, and if one fails, switch it out for one that is working.
```

Comment:
if one fails, switch it out for one that is working -> If current active node moves into failure state then turn on the service in other working node.

Reply:
Did the required changes.